app/assets/javascripts/experiments/table.js

```diff
@@ -226,7 +227,18 @@ var ExperimnetTable = {
         }
       });
 
+      if ($(`.my-module-selector[data-my-module="${id}"]`).prop('checked')) {
+        $(`.my-module-selector[data-my-module="${id}"]`).click();
+      }
+
       return true;
+    };
+
+    $('#modal-edit-module').on('click', 'button[data-action="confirm"]', handleRenameModal);
+
+    $('#modal-edit-module').find('form').submit((e) => {
```

Comment:
Can you please use delegated event and `.on` syntax? It is more readable. And you can chain it to line 237.

Reply:
Hi @aignatov-bio 

Sure, good suggestion!